v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -206,6 +206,15 @@ Please add the property "type" with the value "module" in your package.json to e
       websiteTitle: "Invariant violation",
       websiteDescription: `An internal invariant was violated. This is probably caused by a programming error in Hardhat or in one of the used plugins.
 
+Please [report it](https://github.com/nomiclabs/hardhat/issues/new) to help us improve Hardhat.`,
+      shouldBeReported: true,
+    },
+    NOT_IMPLEMENTED_ERROR: {
```

Comment:
I initially piggy backed on the assertion error. But I suspect it is better for us to be explicit so we can track unimplemented sections of the code base.